snapshot-labs/sx-monorepo
# fix: reset voting power store while switching accounts

### Summary
- While switching accounts on different pages than where we show power, it doesn't reset the voting power of previous account
- Now we reset voting power when ever a wallet is changed or disconnects directly in store
- Removes unwanted resets on composibles

### How to test

1. Connect with a account with which you have some voting power
2. Go to a space you have vp to create proposal ex: http://127.0.0.1:8080/#/base:0x282d013BF31374D0046F04D6a7644d97751ed339/create/
3. Go back to proposals page
4. Switch to a wallet without voting power 
5. Go to editor
6. Before fix: It should allow proposal creation
7. After fix: It should show that you don't have voting power

8. Also all pages where we show voting power
